src/software/ai/hl/stp/tactic/shadow_freekicker_tactic_test.cpp

```diff
@@ -9,11 +9,12 @@
 #include "software/new_geom/util/distance.h"
 #include "software/test_util/test_util.h"
 
-TEST(ShadowFreekickerTacticTest, test_shadow_free_kicker_left_side)
+TEST(ShadowFreekickerTacticTest,
+     test_shadow_free_kicker_left_side_with_enemy_robot_at_middle_left_on_field)
 {
     World world = ::Test::TestUtil::createBlankTestingWorld();
```

Comment:
A quick comment at the start of the test explaining the scenario would be helpful.

Reply:
A quick ascii diagram would potentially be very helpful. This website is very helpful for creating them: http://asciiflow.com/

Reply:
Similar comment goes for the tests below as well.

Reply:
ok, I will comment the code and consider adding ascii diagrams

Reply:
Ascii graph added